FutureNorthants/northants-design-system
# Add library icons

Add the following icons for library service:
- Catalogue
- Computer
- Ebook
- Events
- Member

## Testing
- Checkout this branch and `npm run dev` 
- Visit the iconography docs page and test the new icons work as expected
